lightkurve/interact.py

```diff
@@ -215,6 +228,8 @@ def add_gaia_figure_elements(tpf, fig, magnitude_limit=18):
     pix_scale = 4.0  # arcseconds / pixel for Kepler, default
     if tpf.mission == 'TESS':
         pix_scale = 21.0
+    if tpf.mission == 'Spitzer':
+        pix_scale = 1.22
```

Comment:
Can we get the pixel scale from the TPF FITS header, e.g. one of the standard WCS keywords?  I don't super-like hard-coding it here.